kube/lib/otis/config.libsonnet

```diff
@@ -0,0 +1,29 @@
+{
+  _config+:: {
+    otis: {
+      web: {
+        name: 'web',
+        port: 3000,
+        host: 'otis.kubernetes.hathitrust.org',
+        relative_url_root: '/otis',
+        app_config: {
+          configmap: 'production-config',
+          # path to mount above configmap
+          path: '/usr/src/app/config/settings/production',
+          # key from configmap to use for application config
+          key: 'otis.yml'
+        }
+      },
+      database: {
+        port: 3306,
+        ip: '10.255.8.249',
```

Comment:
I'm not wild about having this here, but I guess it's at least short-lived until consul is available.